src/lib.rs

```diff
@@ -568,13 +579,19 @@ pub struct PerCrateAnalysis {
     def_names: HashMap<String, Vec<Id>>,
     ref_spans: HashMap<Id, Vec<Span>>,
     globs: HashMap<Span, Glob>,
-    impls: HashMap<Id, Vec<Span>>,
+    impls: HashMap<Id, Vec<SpanApi>>,
```

Comment:
Could we add a single bool to the crate, rather than one to each span? If not, I'd prefer to use a tuple here rather than add another struct